.processes/bounty.md

```diff
@@ -0,0 +1,81 @@
+# Bounty Processes
+
+The HOPR Association _tech_ members, _community_ members, and [Project Owner](#legend) have agreed on the following bounty processes.
```

Comment:
Would suggest replacing _community_ with _commercial_

We're not very consistent about the team name, but I think _commercial_ is clearer, especially here when bounty hunters will feasibly come from the HOPR community.

Reply:
I agree but I will keep it community until we define this, since I use community in most of the processes.